tests/test_flux.py

```diff
@@ -136,3 +136,20 @@ def test_angular_momentum_flux():
     jdot1 = silly_angular_momentum_flux(h)
     jdot2 = scri.angular_momentum_flux(h)
     assert np.allclose(jdot1, jdot2, rtol=1e-13, atol=1e-13)
+    
+
+def test_boost_flux():
+    import numpy as np
+    import scri
+    from quaternion import rotate_vectors    
+    
+    h = scri.sample_waveforms.single_mode(ell = 8, m = 5)
+    R = np.quaternion(1,1,1,1).normalized()
+    A = scri.boost_flux(h)
```

Comment:
This line should be:
```
A = h.boost_flux()
```
since your function is subbosed to be an attribute of `WaveformModes`.